client/src/components/LoginUI.jsx

```diff
@@ -22,90 +22,98 @@ export const LoginUI = () => {
 
     try {
       // Send credentials to the server for authentication
-     
-      const response = await axios.post(`${baseurl}/login`, { email, password });
-      if(response.status === 200)  {
-        const token = response.data.token
-        localStorage.setItem('token', token);
-        handleUser(response.data.userId)
-        navigate('/Dashboard');
+
+      const response = await axios.post(`${baseurl}/login`, {
+        email,
+        password,
+      });
+      if (response.status === 200) {
+        const token = response.data.token;
+        localStorage.setItem("token", token);
+        localStorage.setItem("userid", response.data.userId);
+        handleUser(response.data.userId);
+        navigate("/dashboard");
+      } else {
+        alert("Invalid email or password.");
       }
-      
-    else {
-      alert('Invalid email or password.');
+    } catch (error) {
+      console.error(error);
+      alert("An error occurred during authentication.");
```

Comment:
animals

Reply:
Strange, I thought that was from a previous commit that you made earlier, @jordan-carlson99 .

Reply:
that doesnt sound like me -fake news